data/videos.yml

```diff
@@ -3693,3 +3693,30 @@ Seattle Xcoders February 2016:
     title: Continuous Integrations for iOS Apps
     tags: [xcode, build, continuous integration]
     vimeo: 158290425
+NSLondon February 2016:
+  - language: English
+    speakers: [Raimon Lapuente, Gergely Orosz]
+    title: "Sorry Objc: It’s not you, it’s me"
```

Comment:
@chriseidhof Any ideas on this? While it would be nice to identify and fix the problem, if you think fixing is out-of-scope I can change the name to use straight quotation marks `'` instead of proper ones `’`.